cypress/integration/views/repository.view.ts

```diff
@@ -0,0 +1,3 @@
+export const clearRepository = 'button[class="pf-c-button pf-m-link pf-m-inline"]';
+export const confirmClear = 'button[aria-label="confirm"]';
+export const enableInsecureRepository = ".pf-c-switch__toggle";
```

Comment:
This can be a method in utils where you can check if the button is enabled or disabled before clicking it .
Because if the button is already enabled and you click on it will disable it. Right now the code is not checking the existing state of button .

Example : https://github.com/konveyor/tackle-ui-tests/blob/main/cypress/utils/utils.ts#L1171